services/userServices.js

```diff
@@ -0,0 +1,57 @@
+const userDao = require('../models/userDao');
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+const validate = require('../utils/validators');
+const SecretKey = process.env.SECRET_KEY;
```

Comment:
해당 변수가 사용되는 함수 내부로 이동시켜주면 좋을 것 같습니다.

문자열 (상수) 데이터를 할당하는 경우 대문자로 적어주는 것도 하나의 컨벤션입니다.
```suggestion
const SECRET_KEY = process.env.SECRET_KEY;
```